invenio_indexer/cli.py

```diff
@@ -49,16 +49,24 @@ def abort_if_false(ctx, param, value):
 @click.option(
     '--concurrency', '-c', default=1, type=int,
     help='Number of concurrent indexing tasks to start.')
+@click.option('--queue', '-q', default=1, type=str, help='Name of the queue.')
```

Comment:
the default should not be an integer right?

Reply:
Probably `None` (as that's the value you use later to see if it was passed or not).

Reply:
I'd also add an extra clarification, something like "Name of the celery queue used to put the tasks into.", I always felt a bit lost as there's a queue for the data `indexer` and another one for the tasks...